### ISSUE TITLE
Wrong path to in service file

### ISSUE THREAD

**Issue by davidich** [aspnet/Docs#6828]
&gt; ExecStart=/usr/local/bin/dotnet /var/aspnetcore/hellomvc/hellomvc.dll
there is no file dotnet in /usr/local/bin
I guess it should be one of the following:
* /usr/bin/dotnet
* /usr/share/dotnet/dotnet

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cfd9a1f0-c1b6-4443-0700-4c5beafeed03
* Version Independent ID: c22ca7a0-d1d4-200d-fc36-a9798d350c6e
* Content: [Host ASP.NET Core on Linux with Apache](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-apache?view=aspnetcore-2.1&tabs=aspnetcore2x)
* Content Source: [aspnetcore/host-and-deploy/linux-apache.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/host-and-deploy/linux-apache.md)
* Product: **asp.net-core**
* GitHub Login: @spboyer
* Microsoft Alias: **spboyer**

**Comment by cycbluesky** [comment]
Yes, I have no /usr/local/bin/dotnet on CentOS Linux 7. I'm curious why the document has not  corrected untill now.

**Comment by guardrex** [comment]
@cycbluesky Please open a new issue using the **Content feedback** button at the bottom of the topic. You can cross reference this issue (#6828) from the new issue. We don't work closed issues.

**Comment by guardrex** [comment]
Actually, I think the cross-reference is #6827 ... looks like I moved this to the other issue.
